index.js

```diff
@@ -6,6 +6,18 @@ var optionsManager = require('./options-manager');
 
 exports.lintText = function (str, opts) {
 	opts = optionsManager.preprocess(opts);
+
+	if (opts.overrides && opts.overrides.length) {
+		var overrides = opts.overrides;
+		delete opts.overrides;
+
+		var filename = path.relative(opts.cwd || process.cwd(), opts.filename);
```

Comment:
`findApplicableOverrides` needs the path to be relative in order to... find the applicable overrides.

`|| process.cwd()` removed. Thanks. I probably only read enough of options-manager to get it working.